### PR TITLE
Fish factory

### PR DESCRIPTION
**Pull request by SytzeAndr**
I added a fish spawner. This is part of exercise 2.

### CODE REVIEW

**Comment by Exclaminator** [comment]
This looks little like a factory mayve instead solve  the 12 enemy classes, with a strategy pattern.

**Comment by SytzeAndr** [comment]
I think it is very hard to implement the factory pattern exactly the way how it is shown in the slides. Our teacher said during one of his lectures that a design pattern should not be a restriction, but a tool. It could be that I'm wrong, of course, but I advise too take a look inside our google drive under assignment 3, where I put an explenation of the implementation.

**Comment by daanwerf** [comment]
I think it looks good

**Comment by Exclaminator** [comment]
It's fine.

File: src/test/java/lasersharks/FishBotTest.java [hunk 1]
```diff
@@ -15,43 +15,5 @@
  */
 public abstract class FishBotTest extends FishTest {
```
Comment (Exclaminator) [inline src/test/java/lasersharks/FishBotTest.java]: Umh dit leeg laten is niet zo netjes

File: src/main/java/lasersharks/FishBot.java [hunk 2]
```diff
@@ -60,53 +23,4 @@
   public FishBot(Position position, float size, double speed, Direction direction) {
```
Comment (Exclaminator) [inline src/main/java/lasersharks/FishBot.java]: Umh welke responisbility heeft deze class?

**Pull request merged**